### PR TITLE
Feature/antra password reset successful

### PR DESCRIPTION
**Pull request by narendarlgentyala-antra**
Hi Patrick,

I have generated pull request for 3 features such as antra password reset , antra change password and antra password reset successful.

Thanks

### CODE REVIEW

File: projects/antra-ui/src/lib/components/change-password/change-password.component.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+h3 {
```
Comment (Patrick-FRE) [inline projects/antra-ui/src/lib/components/change-password/change-password.component.scss]: do not use tag name directly. Create a class for it.

File: projects/antra-ui/src/lib/models/login.model.ts [hunk 2]
```diff
@@ -1,4 +1,5 @@
 export class Login {
   email: string;
   password: string;
+  confirmPassword: string;
```
Comment (Patrick-FRE) [inline projects/antra-ui/src/lib/models/login.model.ts]: You can create a new Interface like ResetPwdModel.
confirmPassword is not for login.

**Review by Patrick-FRE — CHANGES_REQUESTED** [review]

File: projects/antra-ui/src/lib/models/login.model.ts (hunk 2 above)
Comment (narendarlgentyala-antra) [reply projects/antra-ui/src/lib/models/login.model.ts]: Changes incorporated. Thanks.

File: projects/antra-ui/src/lib/components/change-password/change-password.component.scss (hunk 1 above)
Comment (narendarlgentyala-antra) [reply projects/antra-ui/src/lib/components/change-password/change-password.component.scss]: Changes incorporated. Thanks.

**Pull request merged**